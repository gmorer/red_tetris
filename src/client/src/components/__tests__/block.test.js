import Block from './../block'
import React from 'react';
import renderer from 'react-test-renderer';

test('Some test', () => {
	const component = renderer.create(<Block color="#123456" height="50%" width="50%" />)
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
})