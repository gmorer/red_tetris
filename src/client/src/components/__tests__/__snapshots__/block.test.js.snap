// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some test 1`] = `
<div
  style={
    Object {
      "border": "2px solid #0a1f33",
      "boxSizing": "border-box",
      "height": "50%",
      "width": "50%",
    }
  }
>
  <div
    style={
      Object {
        "height": "100%",
        "overflow": "hidden",
        "width": "100%",
      }
    }
  >
    <div
      style={
        Object {
          "borderColor": "#123456 red red #153e67",
          "borderStyle": "solid",
          "borderWidth": "10em 0em 0em 10em",
          "height": "0px",
          "width": "0px",
        }
      }
    />
  </div>
</div>
`;
